src/js/components/FormField/FormField.js

```diff
@@ -309,6 +314,8 @@ const FormField = forwardRef(
     if (themeBorder.position === 'outer') {
       if (normalizedError && formFieldTheme.error) {
         outerBackground = formFieldTheme.error.background;
+      } else if (focus && formFieldTheme.focus) {
```

Comment:
Similar comment as with `border` above.

Reply:
Currently for `FormField` background there is no default so if none of these checks exists then it would be undefined would you want `formfield` to have a default background prop to fall back on ?

Reply:
no

Reply:
you just. want the same checks as above.. sorry I think I just overthought you comment